apm-agent-core/src/main/java/co/elastic/apm/agent/logging/LoggingConfiguration.java

```diff
@@ -170,6 +170,15 @@ public void assertValid(Boolean value) {
         })
         .buildWithDefault(false);
 
+    private final ConfigurationOption<Boolean> logEcsServiceName = ConfigurationOption.booleanOption()
+        .key("log_ecs_service_name")
+        .configurationCategory(LOGGING_CATEGORY)
+        .tags("added[1.28.0]")
+        .description("Specifies if the agent should set the service name for an ECS compatible logging implementation if it is not explicitly set.\n\n" +
+            "NOTE: Only log4j2-ecs-layout is currently supported")
+        .dynamic(false)
+        .buildWithDefault(false);
```

Comment:
Why did you go for disabling this by default? My instinct would be to always enable or even not making it configurable or making it an internal/undocumented config option.

Reply:
I disabled it by default because other options related to ecs logging are also disabled by default (e.g. log_ecs_reformatting).

Reply:
I am fine to change it